configs/environments/index.ts

```diff
@@ -5,13 +5,12 @@ const parsedEnvs = bootstrap.before()
 const isProd = process.env.NODE_ENV === 'production'
 const env = isProd ? production : development
 
-
 // Preference of environmental variables.
-Object.keys(env.mongo).forEach(key => {
+Object.keys(env.mongo).forEach((key) => {
   env.mongo[key] = process.env[key] || env.mongo[key]
```

Comment:
Maybe we can cover all environment variables, not just `mongo`, or remove these lines of code? (we don't seem to need to declare `mongo` alone)

If we declare environment variables in file `.env`, these variables are assigned to `process.env`. this is expected behavior:

<br />

In the `.env` file
```
//  .env
DB_NAME: '123'
```
And in the `config/environments/production.ts`
```
DB_NAME: '000'
```

<br />

`process.env` at runtime:
```
console.log(process.env.DB_NAME)  // => 123
```

<br />

When used in other files:
```
import { DB_NAME } from './environments'

console.log(Environment.SECRET_NAME) // => 123
```